libs/core/src/components/pds-button/docs/pds-button.mdx

```diff
@@ -143,6 +120,27 @@ Destructive button variant, intended to be used with destructive actions, such a
   <pds-button variant="destructive" icon="trash">Icon</pds-button>
 </DocCanvas>
 
+### Disclosure
+
+Disclosure button variant, intended to be used as a trigger to show or hide content elsewhere on the page.
```

Comment:
Looking at this again, what makes this different than the newly added "CTA" Button? Other than the href

Reply:
@QuintonJason, I am not familiar with the difference of the disclosure. Can you shed any light here?

Reply:
@ju-Skinner the icon is also displayed on the the right side which isn't supported atm with the other variants.

Reply:
Further team discussion is needed around the behavior of the "disclosure" button.